src/views/log-sub-view.cpp

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
❓ avrebbe senso mettere le subview in una loro cartella?
Così le distinguiamo dalle view, che sono schermo intero. Questi sono come dei componenti, dei pezzi di schermo

Reply:
si ma sono solo se fossero tante, per adesso sono poche quindi è equivalente tenerle qui o da un altra parte

Reply:
A me sembra stiano già diventando tante. Io ci faccio già fatica ad orientarmi in sta cartella che mischia le views con le sub-views. Però boh. Alla fine è uguale come dici te quindi chissiene.